packages/donations-app/app/components/home/homeController.js

```diff
@@ -1,19 +1,43 @@
 angular.module('aliceApp')
-  .controller('HomeController', ['ProjectService', 'NotificationService', 'API', '$location', '$http', function (ProjectService, NotificationService, API, $location, $http) {
+  .controller('HomeController', ['ProjectService', 'NotificationService', 'API', '$location', '$http', '$uibModal', 'MODE', function (ProjectService, NotificationService, API, $location, $http, $uibModal, MODE) {
     var vm = this, navHeight = $('#navbar-menu').height();
 
     var loadData = function () {
       vm.contact = {};
       loadProjectsWithCharities();
     };
 
-    var loadProjectsWithCharities = function () {
-      ProjectService.getActiveProjects().then(function (projects) {
-        vm.projects = _.map(projects.data, (project) => {
-          let keyList = ['title', 'code', 'charity', 'lead', 'img'];
+    if (MODE != 'prod') {
```

Comment:
Which mode do we want to use for Vodafone, aren't we going to accept real money?

Reply:
Livia said, we are going to use stage